test/holder.go

```diff
@@ -44,11 +44,8 @@ func (h *Holder) Close() error {
 }
 
 // Reopen closes the holder and instantiates and opens a new holder.
+// note that the hold most be Closed first
```

Comment:
What was the reason for removing the `Close()` from this `Reopen()` method?
With this change it no longer agrees with the original comment.